doc/features/tuning-policies/README.md

```diff
@@ -157,16 +157,175 @@ unexpected error, invoked in the following situations:
     - When the contacted host replies with an error, such as `overloaded`, `isBootstrapping`, `serverError`, etc. In 
     this case, the error is instance of `ResponseError`
 
-The [operation info][OperationInfo], passed as a parameter to the retry policy methods, exposes the `query` and query 
-`options` as properties.
+A default and base retry policy is included, along with `IdempotenceAwareRetryPolicy` that considers query idempotence. The default implementation doesn't use `IdempotenceAwareRetryPolicy`, in order to use it you should wrap the default policy with `IdempotenceAwareRetryPolicy`:
+
+``` js
+ client = new cassandra.Client({
+     contactPoints: contactPoints.split(','),
+     keyspace: keyspace,
```

Comment:
Avoid declaring properties that are optional not directly related to the code same.

Reply:
Thanks